provisioning/miniprov/hare_mp/store.py

```diff
@@ -41,3 +42,26 @@ def __init__(self, url: str):
 
     def _raw_get(self, key: str) -> str:
         return self.conf.get('hare', key)
+
+    def get_machine_id(self) -> str:
+        with open('/etc/machine_id', 'r') as f:
+            machine_id = f.readline().strip('\n')
+            hostname = self.conf.get('hare',
```

Comment:
Let's use `self._raw_get()` instead. When we have a single point where our class interacts to the real confstore, it is simpler to test (mock) and/or override the functionality.

Reply:
:), did use `_raw_get()` before, changed it to `conf.get()`. Yeah, makes sense to use `_raw_get()`.

Reply:
Done.